src/main/java/io/wispforest/gadget/dump/read/handler/MinecraftSupport.java

```diff
@@ -25,7 +24,7 @@ private MinecraftSupport() {
 
     public static void init() {
         PacketUnwrapper.EVENT.register((packet, errSink) -> {
-            if (!Objects.equals(packet.channelId(), CustomPayloadS2CPacket.BRAND)) return null;
+            if (!Objects.equals(packet.channelId(), new Identifier("brand"))) return null;
```

Comment:
The brand packet's identifier is still available as `BrandCustomPayload.ID`

Reply:
https://github.com/wisp-forest/gadget/pull/33/commits/74bc1e207024d9a3e1ecad3cb812145f3207d80c#diff-2ed5c98315bbc10b5bf87ab468eeece32c5323b3f3f6d4b3b5019225f12f4958R28